delfin/drivers/dell_emc/vnx/vnx_block/component_handler.py

```diff
@@ -251,8 +256,10 @@ def list_disks(self, storage_id):
                 hot_spare = disk.get('hot_spare', '')
                 if hot_spare and hot_spare != 'N/A':
                     logical_type = constants.DiskLogicalType.HOTSPARE
+                disk_name = disk.get('disk_name')
+                disk_name = disk_name.replace('  Disk', ' Disk')
```

Comment:
It is recommended to use this way: `' '.join(disk_name.strip().split())` .

Reply:
Modified